plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -236,7 +359,7 @@ global proc int mayaUsdTranslatorExport (string $parent,
         string $skelsAnn = "Exports Maya joints as part of a USD skeleton.";
         string $skinClustersAnn = "Exports Maya skin clusters as part of a USD skeleton.";
         string $blendShapesAnn = "Exports Maya Blend Shapes as USD blendshapes. Requires skeletons to be exported as well.";
-        string $displayColorAnn = "If selected, exports the diffuse color of the geometry’s bound shader as a displayColor primvar on the USD mesh.";
+        string $displayColorAnn = "If selected, exports the diffuse color of the geometry's bound shader as a displayColor primvar on the USD mesh.";

```

Comment:
MAYA-114878 - Use ASCII apostrophe instead of UNICODE one.

Reply:
Do these strings appear in the UI? If so it would be nice if there were moved to the file mayaUSDRegisterStrings.mel.

Reply:
I realize you didn't add these strings, but if we can move them now it would be great.